### PR TITLE
Specify ptable format in CLI procedure

### PR DESCRIPTION
**Pull request by Lennonka**
When the user skips to the CLI procedure, they don't check the example in the WebUI procedure. :)

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13
* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: guides/common/modules/snip_ptable-format.adoc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+The format of the layout must match that for the intended operating system.
+For example, {EL} requires a layout that matches a {client-provisioning-template-type} file, such as:
```
Comment (maximiliankolb) [inline guides/common/modules/snip_ptable-format.adoc:2]:
```suggestion
For example, {client-os} requires a layout that matches a {client-provisioning-template-type} file, such as:
```

File: guides/common/modules/snip_ptable-format.adoc [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
++
+----
+zerombr
+clearpart --all --initlabel
+autopart
+----
++
+See also xref:dynamic-partition-example_{context}[].
```
Comment (maximiliankolb) [inline guides/common/modules/snip_ptable-format.adoc:10]:
```suggestion
For more information, see xref:dynamic-partition-example_{context}[].
```

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by Lennonka** [comment]
Cherry-picked:
- 2c40ae499a..454f3cd83a  3.11 -> 3.11
- f145ae4305..ea37e6d2b2  3.10 -> 3.10
- 0d06ee6498..741506d198  3.9 -> 3.9
- 159ceb54a9..8db0915700  3.8 -> 3.8
- aa82a60148..b047cfe305  3.7 -> 3.7
- 8be7d48b8f..0f2c81d32b  3.6 -> 3.6
- 56f9e92942..9c5af7131e  3.5 -> 3.5

**Pull request merged**